### DISCUSSION TITLE
Can you use Vercel cron jobs with a route that doesn't begin with /api?

### DISCUSSION THREAD

**Discussion by bdlowery** [vercel/vercel#11716] [Help] [answered]
### Question

I have 2 api routes
```
/admin/api/scraper/sites/paylocity
/api/scraper/sites/paylocity
```
They have the exact same code. 

When I try to manually run the cron job from the `/admin/api/scraper/sites/paylocity` i get a **307 Temporary Redirect** error.

When I manually run the cron job from the `/api/scraper/sites/paylocity` route the cron job runs completely fine.

Is there a reason why this would happen? Does the route need to start with `/api` in the vercel.json file?

This is my vercel.json file for reference:
```
{
	"crons": [
		{
			"path": "/admin/api/scraper/sites/paylocity",
			"schedule": "0 0 * * 0"
		}
	]
}
```

**Comment by ignaciodiazb** [comment, marked as answer]
According to the [docs](https://vercel.com/docs/cron-jobs/quickstart#create-or-update-your-vercel.json-file) the path is only required to start with `/` and if the example from you vercel.json works fine, there maybe an issue with how you're handling that request?

If you have protected routes in you app, you also need to add the path of you cron job to your whitelist, so requests won't get blocked by your middleware or auth implementation.

**Reply by rohitpaniker** [reply]
The middleware protecting the routes was causing issue for me and giving 307 error. Thanks for this :)
